### PR TITLE
:tada: New Source: Awtomic

### PR DESCRIPTION
**Pull request by saadatqadri**

### CODE REVIEW

File: airbyte-integrations/connectors/source-awtomic/integration_tests/sample_config.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "fix-me": "TODO"
```
Comment (natikgadzhi) [inline airbyte-integrations/connectors/source-awtomic/integration_tests/sample_config.json:2]: You might want to remove this

File: airbyte-integrations/connectors/source-awtomic/integration_tests/sample_state.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (natikgadzhi) [inline airbyte-integrations/connectors/source-awtomic/integration_tests/sample_state.json:1]: If you're not using the file, remove it. Are all streams full refresh only? If some are incremenental, pleae add test states.

File: airbyte-integrations/connectors/source-awtomic/main.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+
+import sys
+
+from airbyte_cdk.entrypoint import launch
+from source_awtomic import SourceAwtomic
+
+if __name__ == "__main__":
```
Comment (natikgadzhi) [inline airbyte-integrations/connectors/source-awtomic/main.py:11]: `main.py` should invoke `run.py` as in `source-slack`. That's required for the connector to be published to pypi as a package and be available in `pyairbyte`.

File: airbyte-integrations/connectors/source-awtomic/metadata.yaml [hunk 4]
```diff
@@ -0,0 +1,31 @@
+data:
+  allowedHosts:
+    hosts:
+      - localhost # Please change to the hostname of the source.
```
Comment (natikgadzhi) [inline airbyte-integrations/connectors/source-awtomic/metadata.yaml:4]: Not cool, please set the correct hosts.

File: airbyte-integrations/connectors/source-awtomic/metadata.yaml [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  connectorSubtype: api
+  connectorType: source
+  definitionId: a1fb0e0f-9c48-412f-8723-6c45a33027ff
+  dockerImageTag: 0.1.0
+  dockerRepository: airbyte/source-awtomic
+  githubIssueLabel: source-awtomic
+  icon: awtomic.svg
+  license: MIT
+  name: Awtomic
+  releaseDate: TODO
```
Comment (natikgadzhi) [inline airbyte-integrations/connectors/source-awtomic/metadata.yaml:25]: Please set a release date.

File: airbyte-integrations/connectors/source-awtomic/setup.py [hunk 6]
```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+
+from setuptools import find_packages, setup
+
+MAIN_REQUIREMENTS = [
+    "airbyte-cdk~=0.1",
```
Comment (natikgadzhi) [inline airbyte-integrations/connectors/source-awtomic/setup.py:9]: CDK should be unpinned in `pyproject.toml`, to `^0`.

**Review by natikgadzhi — CHANGES_REQUESTED** [review]
Hey @saadatqadri, thank you for the new connector! 👏🏼 

There are a few new requirements (I'm working on writing them down) that we need to get through to be able merge this:
1. Use the latest `baseImage` in `metadata.yaml`. Current is 1.2.0 I believe. 
2. Remove `setup.py` and `requirements`, use Poetry instead. There are several new examples of this, you can use `source-slack` as one.
3. Schemas need to be inlined into the manifest. The easiest way to do this is to open this connector in connector builder and run test reads, this will regenerate schemas in the yaml file itself. Export it and commit it back.
  - This should also update the CDK version to bump from 0.57 to 0.87.6+

Ideally, we would love to have a sandbox account for the source to be able to run connector acceptance tests with records. Do you think you could provide one? If so, let's figure out how you could send it over in a secure way.

**Comment by saadatqadri** [comment]
@natikgadzhi i'll close this and re-create the PR with the recommendations.

**Comment by marcosmarxm** [comment]
@saadatqadri why not make new commits?

**Comment by marcosmarxm** [comment]
@saadatqadri any return about 
> Ideally, we would love to have a sandbox account for the source to be able to run connector acceptance tests with records. Do you think you could provide one? If so, let's figure out how you could send it over in a secure way.

**Comment by saadatqadri** [comment]
@marcosmarxm working on refactoring. Slack ok to send over test API keys

**Comment by marcosmarxm** [comment]
@saadatqadri do you still want to continue this work?

**Comment by saadatqadri** [comment]
@marcosmarxm had some success using the Airbyte builder for this source. closing PR for now.

**Pull request closed without merging**